custom-plugin-decorators/sorting/rule-sort-methods.js

```diff
@@ -0,0 +1,36 @@
+function RuleSortMethods({ order }) {
+  console.log("check method order");
+  return {
+    PathItem: {
+      enter(pathItem, ctx) {
+        // default method sort order, can be changed with an "order" param in config
+        let methodOrder = ["post", "patch", "put", "get", "delete"];
```

Comment:
Should we add head, options?

Reply:
I didn't on the basis that it's configurable so here's a very mainstream starter set and people can work on it from there. Although now you mention it, maybe "get" should be first!